### ISSUE TITLE
[ERROR] [Client-AutoPyTorch:RefitLogger:1] Prediction for lgb failed with run state StatusType.TIMEOUT.

### ISSUE THREAD

**Issue by CHDNY** [automl/Auto-PyTorch#505]
Additional info:
error: Timeout
configuration_origin: traditional
You should provide test set for use best model. use_best_model parameter has been switched to false value.
[WARNING] [2023-09-18 11:30:25,370:Client-AutoPyTorch:RefitLogger:1] Something went wrong while processing the results of catboost.with additional_info: {'opt_loss': {'r2': 0.09571488112562054}, 'duration': 11.153005123138428, 'num_run': 9, 'train_loss': {'r2': 3.774758283725532e-15}, 'test_loss': {'r2': 0.09571488112562054}, 'configuration': 'catboost', 'budget': 50.0, 'configuration_origin': 'traditional'} and status_type: StatusType.SUCCESS. Refer to the log file for more information.
Skipping for now.
[LightGBM] [Warning] min_data_in_leaf is set=3, min_child_samples=20 will be ignored. Current value: min_data_in_leaf=3
[LightGBM] [Warning] feature_fraction is set=0.9, colsample_bytree=1.0 will be ignored. Current value: feature_fraction=0.9
[LightGBM] [Warning] min_data_in_leaf is set=3, min_child_samples=20 will be ignored. Current value: min_data_in_leaf=3
[LightGBM] [Warning] feature_fraction is set=0.9, colsample_bytree=1.0 will be ignored. Current value: feature_fraction=0.9
{'r2': 0.9153034112747092}
|    | Preprocessing   | Estimator   |   Weight |
|---:|:----------------|:------------|---------:|
|  0 | None            | CBLearner   |     0.68 |
|  1 | None            | LGBMLearner |     0.32 |
